package.json

```diff
@@ -307,7 +307,7 @@
     "url-search-params": "^0.10.0",
     "uswds": "1.6.10",
     "vanilla-lazyload": "^16.1.0",
-    "vets-json-schema": "https://github.com/department-of-veterans-affairs/vets-json-schema.git#9692bc6298b6d25ea0b2c950c527e7336f5a3d37",
```

Comment:
@cohnjesse Looks like this points to a commit on [your PR](https://github.com/department-of-veterans-affairs/vets-json-schema/pull/438). To get those changes back, merge your PR and update this to the hash on `master`.